### ISSUE TITLE
Vue wrapper

### ISSUE THREAD

**Issue by verilog15** [dabeng/OrgChart#447]
Have anyone created a Vue wrapper I can check?

**Comment by CaptainCannabis** [comment]
Did you even try?

https://github.com/spiritree/vue-orgchart

**Comment by verilog15** [comment]
@CaptainCannabis Yes I saw it, but it uses the other orgchart project. I'm looking for a wrapper to this project.
